Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift

```diff
@@ -63,9 +63,9 @@ final class GeneralPreferencePane: NSViewController, PreferencePane {
     }
     
     private func setupCheckboxes() {
-        self.hideControlStripCheckbox.state = Defaults[.hideControlStrip]         ? .on : .off
-        self.enableAutomaticUpdates.state   = Defaults[.enableAutomaticUpdates]   ? .on : .off
-        self.launchAtLoginCheckbox.state    = Defaults[.launchAtLogin]            ? .on : .off
+        self.hideControlStripCheckbox.state = Defaults[.hideControlStrip]              ? .on : .off
+        self.enableAutomaticUpdates.state   = Defaults[.enableAutomaticUpdates]        ? .on : .off
+        self.launchAtLoginCheckbox.state    = LoginServiceKit.isExistLoginItems()      ? .on : .off
```

Comment:
👌

Reply:
> 👌

Yes， hahaha
I noticed that and have fixed it